### PR TITLE
elasticsearch-async: add recipe

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: recipes/elasticsearch-async/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - elasticsearch >=5.0.0
+
+test:
+  imports:
+    - elasticsearch_async
+
+about:
+  home: https://github.com/elastic/elasticsearch-py-async
+  license: Apache 2.0
+  license_family: Apache
```
Comment (jakirkham) [inline recipes/elasticsearch-async/meta.yaml]: Is there a license file that we can include in the package?

Comment (nehaljwani) [reply recipes/elasticsearch-async/meta.yaml]: Wasn't part of the source tarball. Added it as part of the recipe now.

Comment (jakirkham) [reply recipes/elasticsearch-async/meta.yaml]: Could you please raise an issue about this upstream?

**Comment by jakirkham** [comment]
Thanks @nehaljwani.

**Pull request merged**